setup.py

```diff
@@ -83,6 +83,7 @@ def get_setup_version(reponame):
         'pooch',
         'scipy',
         'ipywidgets',
+        'rioxarray',
```

Comment:
@Hoxbro could you move `rioxarray` from the `tests` requirements to the `examples` requirements? Previously the tests you touched in this PR were skipped on Python 3.6 as the test suite only installs `-o tests` in this case, and these dependencies don't include any IO packages. I guess we could argue what dependencies should be part of the `tests` dependencies, but I guess for now we could just keep the status quo, at least to unblock this PR.